test/core/end2end/fuzzers/hpack.dictionary

```diff
@@ -4,20 +4,13 @@
 "\x07:status"
```

Comment:
Not directly related to this PR, but we should probably add this file to `.gitattributes` so that github doesn't show diffs by default.

Reply:
It's going to get deleted in a few PR's time, let's let it live until then :)